var/www/html/wp-config.php.ctmpl

```diff
@@ -19,6 +19,12 @@ define( 'DB_PASSWORD', '{{env "MYSQL_PASSWORD"}}' );
 define( 'DB_HOST',     '{{if service "mysql-primary"}}{{with index (service "mysql-primary") 0}}{{.Address}}:{{.Port}}{{end}}{{end}}' );
 
 /**
+ * Handle SSL reverse proxy
```

Comment:
Fix PR'd in https://github.com/jasonpincin/wordpress/pull/2 and https://github.com/jasonpincin/wordpress/pull/3